manifests/vcloud-basic-auth.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: vcloud-basic-auth
+  namespace: kube-system
+data:
+  username: ""
```

Comment:
Could we make this username and password also a parameter that could be replaced by a script? For example, is using BASE64_USERNAME okay, or can does this need to be a string in quotes? If so, we could have "BASE64_USERNAME". I'm thinking about the case if we wanted the CSE serve to pass these parameters

Reply:
CSE server will never have the password. However it could pass the machine user token next.

As of now, if we directly apply the file with "BASE64_USERNAME" etc, k8s will report a failure since it's not a valid base64 encoded value. Hence I have put in the empty value.